lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -1,7 +1,15 @@
 defmodule Mix.Compilers.Elixir do
   @moduledoc false
 
-  @manifest_vsn :v3
+  @manifest_vsn :v4
+
+  import Record
+
+  defrecordp :module, [:module, :kind, :source, :beam, :binary]
+
+  defrecordp :file, [:file, :compile, :runtime]
+
+  defrecordp :source, [:source, :files]
```

Comment:
A little extra whitespace never hurts readability, as long as you keep it reasonable, is my personal philosophy. Just depends where you draw the line personally, I guess. :)